sorts/stooge_sort

```diff
@@ -0,0 +1,23 @@
+def stoogesort(arr, l, h):
```

Comment:
Please be kind to your reader and use better variable names than __l__ and __h__.  Would it be possible to remove l and h and pass in only __arr__?

Reply:
Please insert a doctest of the form:
```python
def stoogesort(arr, l: int, h: int) -> None:
    """
    >>> arr = list(reversed(range(0, 11, 2)))
    >>> stoogesort(arr, 0, len(arr) - 1)
    >>> print(arr)
    [0, 2, 4, 6, 8, 10]
    """
```
and then locally run __python3 -m doctest -v stooge_sort.py__ to ensure that test passes.